web/src/test/resources/springfox.properties

```diff
@@ -0,0 +1 @@
+springfox.documentation.swagger.v2.path:/api-docs
```

Comment:
What's this for?

Reply:
Oop, think that's a holdover from another PR. Deleted.